README.md

```diff
@@ -41,13 +41,13 @@ And that's all, you can connect to your embedded-elastic instance on specified p
 | ------------- | ------------- |
 | `withElasticVersion(String version)` | version of Elasticsearch; based on that version download url to official Elasticsearch repository will be created |
 | `withDownloadUrl(URL downloadUrl)` | if you prefer to download Elasticsearch from different location than official repositories you can do that using this method |
-| `withPortNumber(int portNumber)` | port number on which Elasticsearch will be started |
-| `withClusterName(String clusterName)` | cluster name for created Elasticsearch instance |
-| `withSetting(String name, Object value` | setting name and value as in elasticsearch.yml file |
-| `withMapping(InputStream mapping)`, `withMapping(String mapping)` | JSON with mapping of your index |
-| `withSettings(InputStream settings)`, `withSettings(String settings)` | JSON with settings of your index |
-| `withPlugin(String name, URL urlToDownload)` | plugin that should be installed into Elasticsearch; use multiple times for multiple plugins |
+| `withSetting(String key, Object value)` | setting name and value as in elasticsearch.yml file |
+| `withPlugin(String name, String expression)` | plugin that should be installed into Elasticsearch; treat expression as argument to `./elasticsearch-plugin install` command; use multiple times for multiple plugins |
```

Comment:
I have found some unused imports in differed places, you can optimize imports.
Code looks cool. Good job!